src/LibraryManager.Vsix/UI/Converters/CheckBoxAutomationNameConverter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Converters
+{
+    internal class CheckBoxAutomationNameConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values == null || values.Length != 2 || !(values[0] is string) || !(values[1] is bool))
+            {
+                return null;
+            }
+
+            bool isChecked = (bool)values[1];
+
+            if (isChecked)
+            {
+                return "Checked" + (string)values[0];
```

Comment:
Do these strings need to be localized? (honestly not sure)

Reply:
I am not sure. Tagging @jodavis

Reply:
Yes I think they do, since they'll be read. I'm trying to see if there's a way to access the localized values that would come from the Checkbox by default. (Can you bind value[1] to the Checkbox's automation name, instead of its IsChecked property?)

Otherwise, let's make it localizable, but we won't wait on the loc handback to push the changes.

Reply:
At a minimum, there needs to be a space after "Checked" and "Unchecked", or else they become "Checkedbootstrap"